CODIGO/BabelUI.bas

```diff
@@ -79,6 +79,7 @@ Public Declare Sub LoginAddCharacter Lib "BabelUI.dll" (ByVal name As String, By
 Public Declare Sub LoginSendCharacters Lib "BabelUI.dll" ()
 Public Declare Sub RequestDeleteCode Lib "BabelUI.dll" ()
 Public Declare Sub RemoveCharacterFromList Lib "BabelUI.dll" (ByVal Index As Long)
+Public Declare Function GetTelemetry Lib "BabelUI.dll" (ByRef code As Byte, ByRef DataBuff As Byte, ByVal BuffSize As Long) As Long
```

Comment:
This line is wrong.

it should be.
`Public Declare Function AOT_GetTelemetryResult Lib "AoServerTools.dll" (ByRef Data As Byte, ByVal DataSize As Long, ByVal TelemetryId As Long, ByRef OutBuffer As Byte, ByVal BuffLength As Long) As Long`

Reply:
no this is ok, AoServerTools.dll is a server .dll, on the client the call is inside babel